src/components/balance/modals/ModalTransferAmount.vue

```diff
@@ -219,7 +210,6 @@ export default defineComponent({
       const chain = chainInfo ? chainInfo.chain : '';
       return getProviderIndex(chain);
     });
-    const isH160 = computed(() => store.getters['general/isH160Formatted']);
```

Comment:
can you tell me what is the difference between `const isH160 = computed(() => store.getters['general/isH160Formatted']);` and `const isEthWallet = computed(() => store.getters['general/isEthWallet']);`?

Reply:
isH160: `true` whenever the user want to have an EVM transaction such as H160 to H160 transaction by Ethereum wallet (`false` for SS58 transaction such as dAppsStaking).

isEthWallet: `true` whenever the user connects to the portal by Ethereum wallet